### DISCUSSION TITLE
Packages/com.unity.ugui/api/UnityEngine.EventSystems.PointerEventData

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#260] [Notes]
# Packages/com.unity.ugui/api/UnityEngine.EventSystems.PointerEventData

https://docs.unity3d.com/Packages/com.unity.ugui@latest/index.html?subfolder=/api/UnityEngine.EventSystems.PointerEventData.html

**Comment by radiatoryang** [comment]
To use PointerEventData, you need to add the appropriate interface and implement the event... for example, see [`Selectable.OnPointerEnter`](https://docs.unity3d.com/Packages/com.unity.ugui@latest/index.html?subfolder=/api/UnityEngine.EventSystems.IPointerEnterHandler.html).
